tei/marchese-l-ermenegildo.xml

```diff
@@ -122,7 +122,7 @@
           </personGrp>
           <personGrp xml:id="coro_5" sex="UNKNOWN">
             <name>Coro dell'atto quinto</name>
-          </personGrp>
+          </personGrp>-->
```

Comment:
Let's remove those `personGrp` elements completely instead of just commenting them out.